### ISSUE TITLE
New version 4.2.1. could not open existing environments

### ISSUE THREAD

**Issue by fredshu** [spyder-ide/spyder#14515]
### Issue Report Checklist

* [x ] Searched the [issues page](https://github.com/spyder-ide/spyder/issues?q=is%3Aissue) for similar reports
* [x ] Read the relevant sections of the [Spyder Troubleshooting Guide](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ) and followed its advice
* [x ] Reproduced the issue after updating with ``conda update spyder`` (or ``pip``, if not using Anaconda)
* [x ] Could not reproduce inside ``jupyter qtconsole`` (if console-related)
* [x ] Tried basic troubleshooting (if a bug/error)
    * [x ] Restarted Spyder
    * [x ] Reset preferences with ``spyder --reset``
    * [x ] Reinstalled the latest version of [Anaconda](https://www.anaconda.com/download/)
    * [x ] Tried the other applicable steps from the Troubleshooting Guide
* [x ] Completed the **Problem Description**, **Steps to Reproduce** and **Version** sections below

## Problem Description
After installing new version 4.2.1. it could not run kernels of existing environments. 4.2.0. was working fine.
- However, it managed to open default interpreter from base. 
- Also managed to open interpreter from new environment created for spyder.
- When tried to run kernels of other environments, returned the below error 
PermissionError: [Errno 13] Permission denied: '/home/fd/anaconda3/envs/ok/lib/python3.6/site‑packages/spyder/plugins/ipythonconsole/scripts/conda‑activate.sh'

- Gave permissions to open/read/write all env subdirectories, still did not work
- Final solution: copied conda-activate.sh from /plugins/ipythonconsole/scripts/ from 4.2.0. to 4.2.1. Worked fine with all environments afterwards.

I installed spyder in an existing environment called 'ok' to reproduce the below error.

### What steps reproduce the problem?

1. Install and run new version 4.2.1. in Linux with conda-forge
2. Give permissions to open/read/write all env subdirectories (version 4.2.0 did not require this)
3. Error /usr/bin/env: ‘bash\r’: No such file or directory

### What is the expected output? What do you see instead?
Expected output: Prompt [1] in console. 

Instead got error PermissionError: [Errno 13] Permission denied: '/home/fd/anaconda3/envs/ok/lib/python3.6/site‑packages/spyder/plugins/ipythonconsole/scripts/conda‑activate.sh'

### Paste Traceback/Error Below (if applicable)

```python-traceback
Traceback (most recent call last):
File "/home/fd/anaconda3/envs/ok/lib/python3.6/site‑packages/spyder/plugins/ipythonconsole/plugin.py", line 1428, in create_kernel_manager_and_kernel_client
env=kernel_spec.env)
File "/home/fd/anaconda3/envs/ok/lib/python3.6/site‑packages/jupyter_client/manager.py", line 313, in start_kernel
self.kernel = self._launch_kernel(kernel_cmd, **kw)
File "/home/fd/anaconda3/envs/ok/lib/python3.6/site‑packages/jupyter_client/manager.py", line 220, in _launch_kernel
return launch_kernel(kernel_cmd, **kw)
File "/home/fd/anaconda3/envs/ok/lib/python3.6/site‑packages/jupyter_client/launcher.py", line 131, in launch_kernel
proc = Popen(cmd, **kwargs)
File "/home/fd/anaconda3/envs/ok/lib/python3.6/subprocess.py", line 729, in __init__
restore_signals, start_new_session)
File "/home/fd/anaconda3/envs/ok/lib/python3.6/subprocess.py", line 1364, in _execute_child
raise child_exception_type(errno_num, err_msg, err_filename)
PermissionError: [Errno 13] Permission denied: '/home/fd/anaconda3/envs/ok/lib/python3.6/site‑packages/spyder/plugins/ipythonconsole/scripts/conda‑activate.sh'

```

## Versions

* Spyder version: 4.2.1
* Python version: 3.6.12
* Qt version: 5.12.6
* PyQt version: 5.12.3
* Operating System name/version: Linux 5.4.0.58-generic

### Dependencies

```
PASTE DEPENDENCIES HERE
```

# Mandatory:
atomicwrites >=1.2.0            :  1.3.0 (OK)
chardet >=2.0.0                 :  3.0.4 (OK)
cloudpickle >=0.5.0             :  1.5.0 (OK)
diff_match_patch >=20181111     :  20181111 (OK)
intervaltree >=3.0.2            :  3.0.2 (OK)
IPython >=7.6.0                 :  7.16.1 (OK)
jedi =0.17.2                    :  0.17.2 (OK)
jsonschema >=3.2.0              :  3.0.2 (NOK)
keyring >=17.0.0                :  21.0.0 (OK)
nbconvert >=4.0                 :  5.6.1 (OK)
numpydoc >=0.6.0                :  0.9.2 (OK)
parso =0.7.0                    :  0.7.0 (OK)
pexpect >=4.4.0                 :  4.8.0 (OK)
pickleshare >=0.4               :  0.7.5 (OK)
psutil >=5.3                    :  5.4.8 (OK)
pygments >=2.0                  :  2.6.1 (OK)
pylint >=1.0                    :  2.4.4 (OK)
pyls >=0.36.2;<1.0.0            :  0.36.2 (OK)
pyls_black >=0.4.6              :  0.4.6 (OK)
pyls_spyder >=0.3.0             :  0.3.0 (OK)
qdarkstyle >=2.8                :  2.8.1 (OK)
qtawesome >=0.5.7               :  0.6.1 (OK)
qtconsole >=5.0.1               :  4.6.0 (NOK)
qtpy >=1.5.0                    :  1.9.0 (OK)
rtree >=0.8.3                   :  0.9.7 (OK)
setuptools >=39.0.0             :  49.6.0 (OK)
sphinx >=0.6.6                  :  2.3.1 (OK)
spyder_kernels >=1.10.1;<1.11.0 :  1.10.1 (OK)
textdistance >=4.2.0            :  4.2.0 (OK)
three_merge >=0.1.1             :  0.1.1 (OK)
watchdog >=0.10.3               :  0.9.0 (NOK)
xdg >=0.26                      :  0.26 (OK)
zmq >=17                        :  19.0.2 (OK)

# Optional:
cython >=0.21                   :  0.29.3 (OK)
matplotlib >=2.0.0              :  2.0.2 (OK)
numpy >=1.7                     :  1.19.2 (OK)
pandas >=1.1.1                  :  1.1.4 (OK)
scipy >=0.17.0                  :  1.5.2 (OK)
sympy >=0.7.3                   :  None (NOK)

**Comment by ccordoba12** [comment]
Hey @fredshu, thanks for reporting. This is a known error and it seems to only happen with our conda-forge packages. We'll try to fix it as soon as we can.

**Comment by ajland** [comment]
I was able to fix this issue by navigating to the path in the terminal. Then, once in the folder containing the conda-activate.sh script, I typed:

`chmod +x conda-activate.sh`

This grants everyone execute permissions for this file. Before doing this, if you type `ls -l conda-activate.sh`, you'll see that no one has execute permission. After doing this, you'll see that everyone has execute permission. You should then be able to launch spyder and see that all is normal.

**Comment by ccordoba12** [comment]
@ajland, that error was introduced in 5.5.2  but immediately fixed in 5.5.3, already released.
